packages/card-support/src/components/DownloadClientData/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,173 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/DownloadClientData renders correctly: card 1`] = `
```

Comment:
should this snapshot be broken down?

Reply:
@p-netm It's a snapshot of the props. From my end, I am unable to break it, any ideas?

Reply:
we can have a snapshot for props only and for checking that if it is rendering correctly by finding the component or data that is rendered on success then then checking its lenght or other way would be to use expect match